src/access_nri_intake/data/__init__.py

```diff
@@ -1,9 +1,11 @@
 # Copyright 2023 ACCESS-NRI and contributors. See the top-level COPYRIGHT file for details.
 # SPDX-License-Identifier: Apache-2.0
 
-import os
+import importlib
 
 import intake
 
-_here = os.path.abspath(os.path.dirname(__file__))
-data = intake.open_catalog(os.path.join(_here, "catalog.yaml")).access_nri
+data = intake.open_catalog(
+    importlib.resources.files("access_nri_intake").joinpath("data/catalog.yaml")
+    # os.path.join(_here, "catalog.yaml")
+).access_nri
```

Comment:
As this code is duplicated would it be a good idea to make it a function that can be used in the two locations? Would that help with testing?

Reply:
It could, although I'm going to have to do some beard-stroking as I implement it. This block loads the existing catalog; the block referenced above (cli.py, L213-223) updates the stored catalog. I'll just need to make sure whatever helper I come up with has actually been loaded in both of those situations.

Reply:
I've updated to a helper function with a related test, but I need to think further about how to test:

- The catalog update in `cli.py`, and
- The `intake` catalog generation here. This step looks for `/g/data` access, so I'm going to have to come up with a way to mock that.

Reply:
I'll resolve this particular comment thread as the importlib call has been factored out into its own testable function, but I'll repeat the above comments about testing during the next review pass.